Sources/BraveShared/Javascript/JSValueExtensions.swift

```diff
@@ -1,32 +0,0 @@
-// This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Those deletions are not part of this ticket right?
It's fine to remove them here, just want to be sure

Reply:
Technically not, but thats where `ErrorExtensions.siwft` was living so just got rid of the whole folder instead